compiler/src/dmd/expression.h

```diff
@@ -52,7 +52,7 @@ void expandTuples(Expressions *exps, Identifiers *names = nullptr);
 StringExp *toUTF8(StringExp *se, Scope *sc);
 Expression *resolveLoc(Expression *exp, const Loc &loc, Scope *sc);
 MATCH implicitConvTo(Expression *e, Type *t);
-Expression *toLvalue(Expression *_this, Scope *sc);
+Expression *toLvalue(Expression *_this, Scope *sc, const char* action);
```

Comment:
fwiw, now that `toLvalue` is a free function it is no used by LDC. If it is unused by GDC we can make it `extern(D)` and remove it from the headers